internal/criocli/criocli.go

```diff
@@ -832,10 +835,6 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			Name:  "enable-nri",
 			Usage: fmt.Sprintf("Enable NRI (Node Resource Interface) support. (default: %v)", defConf.NRI.Enabled),
 		},
-		&cli.StringFlag{
-			Name:  "nri-config-file",
-			Usage: fmt.Sprintf("NRI configuration file to use. (default: %q)", defConf.NRI.ConfigPath),
-		},
```

Comment:
We should deprecate the flag by keeping it, throwing an error or/and falling back to `nri-plugin-config-dir`.

Reply:
we could also mark deprecated in 1.26.z and drop here--since it's a experimental feature I am not super worried about compatibility. Though, if you feel strongly @saschagrunert we can deprecate in 1.27 instead

Reply:
Hm, yeah sounds good to me, too. :+1:

Reply:
@haircommander I've tested the #6645 by @Tal-or rebased on top of this and unit tests do pass now.